hyperspy/_signals/signal1d.py

```diff
@@ -1467,6 +1467,9 @@ def estimate_peak_width(
         background must be previously substracted.
         The estimation is performed by interpolation using cubic splines.
 
+        Warning: `parallel=True` is not currently supported on
```

Comment:
Use the `Raises` section of numpydoc instead and update/append the `PARALLEL_ARG` string mentioning the windows issue?